### ISSUE TITLE
Dependency on psr/cache:^1

### ISSUE THREAD

**Issue by aleksblendwerk** [ScriptFUSION/Porter#73]
I just wanted to take Porter for a quick spin, created a new Symfony project and tried to require the Porter package, resulting in this error:

```
scriptfusion/porter 7.0.0 requires psr/cache ^1 -> found psr/cache[1.0.0, 1.0.1] but the package is fixed to 3.0.0 (lock file version)
```

Is an update feasible, best for `psr/container` as well?

**Comment by Bilge** [comment]
Have you tried not fixing the package to v3? Sounds like you ran `composer require` without the `-w` or `-W` flags.

Try `composer req scriptfusion/porter -W`.

If you absolutely need a newer version of `psr/cache` for some reason, we should be able to add compatibility for the newer versions without much difficulty.

**Comment by aleksblendwerk** [comment]
Thanks for the quick response but requiring with `-w` or `-W` did not help.

`symfony/cache` 6.0.0 and up requires `psr/cache: ^2.0|^3.0`.
`symfony/service-contracts` 3.0.0 and up requires `psr/container: ^2.0`.

Like I said I just wanted to give Porter a shot for a side project, so this is anything but urgent, just thought I'd report it.

The new project I created and where I tried this is based on Symfony 6.2, by the way. 
Basically just the result of `symfony new test --webapp`.

**Comment by Bilge** [comment]
Seems like you also have a conflicting dependency on `psr/container`, in addition to `psr/cache`. Will see what we can do about both. 

Unfortunately our [Symfony Quickstart](https://github.com/ScriptFUSION/Porter/blob/master/docs/Quickstart%20Symfony.md) has only been tested with v5, not yet compatible with v6.

**Comment by Bilge** [comment]
I have just released Porter 🏷️ [7.0.1](https://github.com/ScriptFUSION/Porter/releases/tag/7.0.1) which should resolve your issues.

**Comment by aleksblendwerk** [comment]
It installs fine now after I did these changes to the generated Symfony project:

`composer require symfony/cache-contracts:^2`
`composer require psr/cache:^2`

`symfony/cache-contracts` 3.0.0 and up requires `psr-cache: ^3.0`, I didn't spot this one earlier, sorry.

But `composer req scriptfusion/porter` works now, thanks a lot!

**Comment by Bilge** [comment]
Porter 🏷️ [7.0.2](https://github.com/ScriptFUSION/Porter/releases/tag/7.0.2) adds compatibility with `psr/cache` v3.
